packages/stark-ui/src/modules/table/components/table.component.ts

```diff
@@ -54,58 +54,59 @@ const componentName: string = "stark-table";
 /* tslint:enable */
 export class StarkTableComponent extends AbstractStarkUiComponent implements OnInit, AfterContentInit, AfterViewInit, OnChanges {
 	/**
-	 * HTML id of the table
+	 * Array of StarkColumnProperties objects which define the columns of the data table.
```

Comment:
You can improve this JSDoc by giving the link to the `StarkColumnProperties` like this:

`Array of {@link StarkColumnProperties} objects which define the columns of the data table.`

Reply:
done